### ISSUE TITLE
Is it possible to Show only year in Calendar Picker

### ISSUE THREAD

**Issue by mitulshah96** [stephy/CalendarPicker#180]
Is it possible to Show only year in Calendar Picker ?

**Comment by peacechen** [comment]
Not currently.  Without any dates, would the user only select a year?

You're welcome to submit a PR to add this feature.

**Comment by mitulshah96** [comment]
Yes there should be something mode as a prop which identify what type of calendar a user want to see for example mode={date || year || datetime || month }

something like this https://www.npmjs.com/package/react-year-picker

**Comment by peacechen** [comment]
A year picker has been added in commit 67a03746500896ab2a8c90421fde71fa5f00d34b.

You can test this by pulling directly from this repo or running the example project. To pull directly from this repo, use this in your package.json:
```
dependencies: {
    "react-native-calendar-picker": "https://github.com/stephy/CalendarPicker",
}
```

The year selector view is displayed by pressing the year in the header.  Currently there's no prop to control whether day, month, or year selector views are shown by default.  Another consideration is whether that prop would restrict the view to only the year selector (or month, or day).  This should be a straight-forward prop to add.  Please submit a PR and I'll review promptly.
